src/KM_Controls.pas

```diff
@@ -4610,7 +4628,7 @@ function TKMListBox.KeyDown(Key: Word; Shift: TShiftState): Boolean;
                 end;
     else        begin
                   if Assigned(OnKeyDown) then
-                    Result := OnKeyDown(Key, Shift);
+                    Result := OnKeyDown(Self, Key, Shift);

```

Comment:
Doesn't this mean fOnKeyDown will fire twice because we already called "inherited KeyDown" (which will call fOnKeyDown too)? Same for other controls like TKMColumnBox etc.